src/plugins/invocables/report/pahmaUoCbyStudentVisits/fields.js

```diff
@@ -0,0 +1,66 @@
+import { defineMessages } from 'react-intl';
+
+export default (configContext) => {
+  const {
+    CompoundInput,
+    AutocompleteInput,
+    DateInput,
+  } = configContext.inputComponents;
+
+  const {
+    configKey: config,
+  } = configContext.configHelpers;
+
+  return {
+    params: {
+      [config]: {
+        view: {
+          type: CompoundInput,
+        },
+      },
+      StaffName: {
+        [config]: {
+          messages: defineMessages({
+            name: {
+              id: 'field.pahmaUoCbyStudentVisits.StaffName.name',
+              defaultMessage: 'Staff Name',
```

Comment:
Hey Lam! Forgot to mention this, but by convention we use sentence case for messages. Please change to "Staff name"

Reply:
Fixed. Thank you!